src/assets/i18n/custom-translations.json

```diff
@@ -178,6 +178,6 @@
   "lineListingSharedDisease": "Shared disease",
   "lineListingSharedLocation": "Shared location",
   "headingUserProfile": "Role name",
-  "confirmSetMissingGeoCoordinates": "Also replace existing coordinates",
-  "warningSetMissingGeoCoordinates": "This might replace coordinates which were intentionally set differently!"
+  "confirmSetMissingGeoCoordinates": "Also replace exising coordinates",
```

Comment:
you've overwrite fixes for a different task

Reply:
corrected